developer/pages-router/installation.mdx

```diff
@@ -96,18 +96,24 @@ Next.js plugins are configured in the project's next.config.js file by wrapping
   </Step>
   <Step title="Set up a custom Document">
   
-Create a [custom Document](https://nextjs.org/docs/pages/building-your-application/routing/custom-document) named `_document.ts` and export `Document` from `@makeswift/runtime/next`. 
-The Makeswift custom Document handles styles during server-side rendering and using [Preview Mode](https://nextjs.org/docs/pages/building-your-application/configuring/preview-mode) when opening your pages in the Makeswift builder.
+The Makeswift [custom Document](https://nextjs.org/docs/pages/building-your-application/routing/custom-document) handles styles during server-side rendering and using [Preview Mode](https://nextjs.org/docs/pages/building-your-application/configuring/preview-mode) when opening your pages in the Makeswift builder.
+
+Create the file `src/pages/_document.ts` and export `Document` from `@makeswift/runtime/next/document`:
```

Comment:
Swapped the sentence order to improve the flow/match the other steps, fixed the import paths:

<img width="814" alt="improved flow" src="https://github.com/user-attachments/assets/54d170e4-6720-46d8-bcf1-dbb7804fbb03" />